playbooks/files/rax-maas/plugins/ironic_conductor_check.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+import psutil
+
+from maas_common import get_auth_ref
+from maas_common import metric_bool
+from maas_common import print_output
+
+
+def check(auth_ref, args):
+    name = "ironic-conductor_status"
+    for proc in psutil.process_iter():
+        if 'ironic-conducto' in proc.name():
```

Comment:
@briancurtin:  It actually is not a misspelling.  I run this script on  beta environment  and the ironic-coductor process's name is indeed **ironic-conducto**.   Weird as it is I agree.

See this snippet:
https://gist.github.com/tonytan4ever/51ee331b999518f6d1aaa4b3a0c358e6